tests/SmileIdentityCoreTest.php

```diff
@@ -82,7 +82,7 @@ public function testSubmitJobUploadsZipFileWhenReturnJobStatusIsFalse(): void
             "partner_params" => [
                 "job_id" => "php-job-id-04",
                 "user_id" => "php-user-18",
-                "job_type" => 1
+                "job_type" => JobType::BIOMETRIC_KYC
```

Comment:
Nit: Can we do the same thing for image type? it's okay if it's in a different pr